x-pack/plugins/security_solution/server/endpoint/routes/actions/isolation.ts

```diff
@@ -157,22 +157,18 @@ export const isolationRequestHandler = function (
       });
     }
 
-    const commentLines: string[] = [];
-
-    commentLines.push(`${isolate ? 'I' : 'Uni'}solate action was sent to the following Agents:`);
-    // lines of markdown links, inside a code block
-
-    commentLines.push(`${agentIDs.map((a) => `- [${a}](/app/fleet#/agents/${a})`).join('\n')}`);
-    if (req.body.comment) {
-      commentLines.push(`\n\nWith Comment:\n> ${req.body.comment}`);
-    }
-
+    // TODO: This sets the comment type, we'll need to use our new type.
     caseIDs.forEach(async (caseId) => {
       (await endpointContext.service.getCasesClient(req)).attachments.add({
         caseId,
         comment: {
-          comment: commentLines.join('\n'),
-          type: CommentType.user,
+          type: CommentType.actions,
+          comment: req.body.comment || '',
+          actions: {
+            endpointId: agentIDs[0],
```

Comment:
this seems.. wrong? First, it's actually agent IDs, not Endpoint IDs (even though they may be technically becoming the same value soon).

And only grabbing the first one seems like we're barreling towards a miscommunication

Reply:
i think the key name should be better -- maybe endpointAgentId? -- also this was just a placeholder to try to get a new comment type saved